lib/consts/consts.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // Version contains the current semantic version of k6.
-const Version = "0.45.0"
+const Version = "0.46.0"
```

Comment:
@oleiade Based on the task list https://github.com/grafana/k6/issues/3138. This change should be in a dedicated PR. I think @mstoykov suggested it, I don't remember exactly why, I guess it is for being sure to preserve an atomic commit for it.

What I care is that we follow the task list so I would like to adjust the template if we want to keep them together or split the PRs if we want to follow the task list.

Reply:
You're right, I realized that after the fact indeed.
I'll move it out 👍🏻

Reply:
> I guess it is for being sure to preserve an atomic commit for it. 


Yeah, predominantly because this PR (the release notes) usually gets a lot of edits, and it is way easier to just push "squash and merge" at the end.

But if you also have the commit for bumping v0.46.0 that will be squashed as well. 

Additionally, I am of the opinion that there are no problems for the release notes for a version to be in master, while the version is still not bumped to. The other way around is IMO less ideal - having the version bump without release notes.

So my preference is literally to have this (PR) merged and then make a new PR that is just the bump and merge and tag it as one at the moment we need to release.